src/ngResource/resource.js

```diff
@@ -702,12 +702,13 @@ angular.module('ngResource', ['ng']).
               return $q.reject(response);
             });
 
-            promise.finally(function() {
+            promise = promise.finally(function(response) {
```

Comment:
The `.finally` callback doesn't take any arguments and the result is ignored unless it's a promise (and even when it's a promise the result of that promise is ignored). I think this arg and the return can be dropped.